recipes/yaclib/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.scm import Version
+from conan.tools.files import copy, get, export_conandata_patches, apply_conandata_patches
+from conan.tools.layout import cmake_layout
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.52.0"
+
+
+class YACLibConan(ConanFile):
+    name = "yaclib"
+    description = "lightweight C++ library for concurrent and parallel task execution"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/YACLib/YACLib"
+    license = "MIT"
+    topics = "c++", "async", "parallel", "concurrency"
+    settings = "os", "arch", "compiler", "build_type"
+
+    _yaclib_flags = {
+        "warn": [True, False],
+        "asan": [True, False],
+        "tsan": [True, False],
+        "ubsan": [True, False],
+        "lsan": [True, False],
+        "memsan": [True, False],
+        "coverage": [True, False],
+        "coro": [True, False],
+        "disable_futex": [True, False],
+        "disable_unsafe_futex": [True, False],
+        "disable_symmetric_transfer": [True, False],
+        "disable_final_suspend_transfer": [True, False],
+    }
+
+    _yaclib_faults = {
```

Comment:
Typo?
```suggestion
    _yaclib_fault = {
```
Btw only one value can be active in the same time